pyinventorymgr/sync_inventory_items.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python -u
+
+import sys
+import dbus
+import getopt
+import subprocess
+import obmc_system_config as System
+
+INV_DBUS_NAME = 'org.openbmc.Inventory'
+INV_INTF_NAME = 'org.openbmc.InventoryItem'
+NET_DBUS_NAME = 'org.openbmc.NetworkManager'
+NET_OBJ_NAME = '/org/openbmc/NetworkManager/Interface'
+CHS_DBUS_NAME = 'org.openbmc.control.Chassis'
+CHS_OBJ_NAME = '/org/openbmc/control/chassis0'
+PROP_INTF_NAME = 'org.freedesktop.DBus.Properties'
+
+FRUS = System.FRU_INSTANCES
+
+# IEEE 802 MAC address mask for locally administered.
+# This means the admin has set the MAC and is no longer
+# the unique number set by the device manufacturer.
+MAC_LOCALLY_ADMIN_MASK = 0x20000000000
+
+# Pop the command name                               
+sys.argv.pop(0)
+
+# Get the inventory dbus path based on the requested fru
+def get_inv_obj_path(fru_type, fru_name):
+    obj_path = ''
+    for f in FRUS.keys():
+       if (FRUS[f]['fru_type'] == fru_type and
+            f.endswith(fru_name)):
+            obj_path = f.replace("<inventory_root>",System.INVENTORY_ROOT)
```

Comment:
This appears to work but pep8 says:

E125 continuation line does not distinguish itself from next logical line

Otherwise, the whole commit LGTM

can you give it a run?